src/main/handlers/media/transcriptionEngines/voskTranscribeFunction.ts

```diff
@@ -0,0 +1,28 @@
+import { MapCallback, PartialWord } from 'sharedTypes';
+import fs from 'fs';
+import path from 'path';
+import { TranscriptionFunction } from '../transcribeTypes';
+
+interface VoskWord {
+  end: number;
+  start: number;
+  word: string;
+}
+
+const voskAdaptor: MapCallback<VoskWord, PartialWord> = (result) => ({
+  word: result.word,
+  duration: result.end - result.start,
+  startTime: result.start,
+});
+const voskTranscribeFunction: TranscriptionFunction = async (project) => {
```

Comment:
I'm guessing this is where I'll come in with the vosk code?

Reply:
Yep, just replace the readFileSync function with the real vosk transcription function if it's outputting a string

Reply:
Too easy, thanks